src/sdk/pynni/nni/nas/benchmark/nasbench201/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .model import Nb201ComputedStats, Nb201IntermediateStats, Nb201RunConfig
+from .shortcuts import query_nb201_computed_stats
```

Comment:
what is the meaning of `shortcuts`?

Reply:
I followed the practice from some other projects. Do you have a better name in your mind?

Reply:
query.py